presto-native-execution/presto_cpp/main/common/Configs.cpp

```diff
@@ -25,12 +28,135 @@ namespace fs = std::experimental::filesystem;
 
 namespace facebook::presto {
 
+namespace {
+
+void checkIncomingSystemProperties(
+    const std::unordered_map<std::string, std::string>& values) {
+  static const std::unordered_set<std::string_view> kSupportedSystemProperties{
```

Comment:
@spershin If we introduce a new system config, we need to update this map for the checking to work?

Reply:
Yes. We can think on how to make it more convenient.
For now, add it to the map.

Reply:
The issue is if we forget to add here :(

Reply:
@spershin There is another issue here. Many properties are not defined in OSS, but defined inside the classes that inherit PrestoServer. Such properties (e.g. storage specific options) will be printed as not used, but actually they are used. We need to extend this capability so that classes that inherit PrestoServer can also push properties for checking.

Reply:
@amitkdutta Then we need to decide if config.properties is a dump for everything or it is a well established set of config properties that are declared somewhere.